tuf/api/metadata.py

```diff
@@ -676,15 +676,11 @@ def __init__(
         threshold: int,
         unrecognized_fields: Optional[Mapping[str, Any]] = None,
     ):
-        keyids_set = set(keyids)
-        if len(keyids_set) != len(keyids):
-            raise ValueError(
-                f"keyids should be a list of unique strings,"
-                f" instead got {keyids}"
-            )
+        if len(set(keyids)) != len(keyids):
+            raise ValueError(f"Nonunique keyids: {keyids}")
```

Comment:
Small nit, feel free to ignore
```suggestion
            raise ValueError(f"Non-unique keyids: {keyids}")
```